en/yohane.md

```diff
@@ -2,33 +2,33 @@
-- `!help` - Memberikan daftar bantuan
-- `!subscribe pp [options]` - Untuk menampilkan status pp seperti limit dan notifikasi jika sudah selesai [submit score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options : `set`,`get`,`unset`,`toggle`)
-- `!boardshow [options]` - [Experimental] Untuk menyembunyikan diri dari leaderboard (options : `others`,`normal`)
-- `!report` - Jika kalian menemukan cheater, silahkan gunakan command ini.
-- `!faq list` - Daftar hal yang sering dibutuhkan dalam chat.
+- `!roll [number]` - Returns random number
+- `!help` - Shows a list of all commands
+- `!subscribe pp [options]` - Notifies about the pp status such as limit after [submitting a score](https://cdn.discordapp.com/attachments/265909019976138754/801454622346444860/unknown.png). (options: `set`,`get`,`unset`,`toggle`)
+- `!boardshow [options]` - [Experimental] Hide yourself from the leaderboard (options: `others`,`normal`)
+- `!report` - If you find a cheater, please use this command.
+- `!faq list` - Returns a list of things that are often needed in chat.
 
 ### Multiplayer commands
 
-Semua command yang ditampilkan disini hanya yang secara umum atau membutuhkan penjelasan. Mohon gunakan `!mp help` apabila bingung dengan command yang tidak terdaftar.
+These are general commands for the use in multiplayer rooms. Please use `!mp help` to show more commands.
 
-- `!mp help` - Untuk menampilkan command multiplayer selengkapnya.
-- `!mp make [name]` - Membuat room multi
-- `!mp close` - Menutup room multi
-- `!mp checkperm` - Menampilkan permission level pada pengirim.
-- `!mp listref` - Menampilkan referee pada room.
-- `!mp addref <username>` - Menambahkan user menjadi referee pada room.
-- `!mp rmref <username>` - Mengeluarkan user dari referee pada room.
-- `!mp start [timer]` - Membuat semua pemain siap secara otomatis dan memulai permainan setelah waktu habis (Default: `0`). Dapat dilakukan untuk solo-play pada multi room.
-- `!mp abort` - Untuk membatalkan permainan tersebut. *Hal ini berbeda dengan menggunakan `Escape`-key pada solo-play*
+- `!mp help` - Shows a list of all multiplayer commands.
+- `!mp make [name]` - Creates a multiplayer room.
+- `!mp close` - Closes the current multiplayer room.
+- `!mp checkperm` - Returns the permission level of the sender.
+- `!mp listref` - Displays a list of referees in the room.
+- `!mp addref <username>` - Adds user as a referee in the room.
+- `!mp rmref <username>` - Removes user as referee in the room.
+- `!mp start [timer]` - Sets all players to ready automatically and starts the game after a timeout (Default: `0`). Can be used to play solo in a multiplayer room.
+- `!mp abort` - Aborts the game. *This is different from using the `Escape`-key in solo*
 
-Note: untuk semua operasi menggunakan Username Lookup, dapat menggunakan `#<UserID>` sebagai penggantinya.
+Note: For every operation using the username lookup, you can use `#<UserID>` instead.
```

Comment:
```suggestion
Note: For every operation that use Username Lookup, you can use `#<UserID>` instead.
```

Reply:
This is not English

Reply:
Does it need a quotation mark for the command Username Lookup? like
```suggestion
Note: For every operation that uses 'Username Lookup' can use `#<UserID>` instead.

Reply:
We can do that, but I wouldn't say it's necessary. Whatever you prefer.

Reply:
Ah, the original file in ID doesn't have quotation, but it does capitalize the commands, so maybe just this.
```suggestion
Note: For every operation using the Username Lookup, you can use `#<UserID>` instead.
```

Reply:
it is already changed in the latest commit

Reply:
Oh sorry, nevermind then